<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Starting Scenario Create DashBoard if there is no dashboards available to the user<br/>]]>
    </line>
    <line>
      <![CDATA[**************************************************************************************************<br/>]]>
    </line>
    <line>
      <![CDATA[Ending Scenario Create DashBoard if there is no dashboards available to the user<br/>]]>
    </line>
    <line>
      <![CDATA[**************************************************************************************************<br/>]]>
    </line>
    <line>
      <![CDATA[Starting Scenario verify signin page<br/>]]>
    </line>
    <line>
      <![CDATA[**************************************************************************************************<br/>]]>
    </line>
    <line>
      <![CDATA[Ending Scenario verify signin page<br/>]]>
    </line>
    <line>
      <![CDATA[**************************************************************************************************<br/>]]>
    </line>
    <line>
      <![CDATA[Starting Scenario Create User and Dashboard<br/>]]>
    </line>
    <line>
      <![CDATA[**************************************************************************************************<br/>]]>
    </line>
    <line>
      <![CDATA[Ending Scenario Create User and Dashboard<br/>]]>
    </line>
    <line>
      <![CDATA[**************************************************************************************************<br/>]]>
    </line>
  </reporter-output>
  <suite started-at="2019-12-16T13:15:18Z" name="Sample Test Suite" finished-at="2019-12-16T13:16:10Z" duration-ms="51853">
    <groups>
      <group name="cucumber">
        <method signature="AbstractTestNGCucumberTests.feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:runnerFile.RunnerFile@69a10787]" name="feature" class="cucumber.api.testng.AbstractTestNGCucumberTests"/>
      </group> <!-- cucumber -->
    </groups>
    <test started-at="2019-12-16T13:15:18Z" name="Sample Test Cases" finished-at="2019-12-16T13:16:10Z" duration-ms="51853">
      <class name="runnerFile.RunnerFile">
        <test-method is-config="true" signature="setUpClass()[pri:0, instance:runnerFile.RunnerFile@69a10787]" started-at="2019-12-16T13:15:18Z" name="setUpClass" finished-at="2019-12-16T13:15:18Z" duration-ms="230" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpClass -->
        <test-method signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:runnerFile.RunnerFile@69a10787]" started-at="2019-12-16T13:15:18Z" name="feature" description="Runs Cucumber Feature" data-provider="features" finished-at="2019-12-16T13:15:46Z" duration-ms="27526" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Trello Create Dashboard For the first time]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.AssertionError]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Caused by: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at pageObject.DashBoardPagePO.verifyPage(DashBoardPagePO.java:43)
	at stepDefinition.DashBoardPage.user_is_on_Dashboard_page_with_title(DashBoardPage.java:35)
	at âœ½.Given User is on Dashboard page with title "Create Your Board | Trello"(TrelloCreateDashBoard.feature:14)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
            <line>
              <![CDATA[Starting Scenario Create DashBoard if there is no dashboards available to the user<br/>]]>
            </line>
            <line>
              <![CDATA[**************************************************************************************************<br/>]]>
            </line>
            <line>
              <![CDATA[Ending Scenario Create DashBoard if there is no dashboards available to the user<br/>]]>
            </line>
            <line>
              <![CDATA[**************************************************************************************************<br/>]]>
            </line>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method signature="feature(cucumber.api.testng.CucumberFeatureWrapper)[pri:0, instance:runnerFile.RunnerFile@69a10787]" started-at="2019-12-16T13:15:46Z" name="feature" description="Runs Cucumber Feature" data-provider="features" finished-at="2019-12-16T13:16:09Z" duration-ms="23814" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Trello Create User Login]]>
              </value>
            </param>
          </params>
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.AssertionError]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError
	at cucumber.api.testng.TestNGCucumberRunner.runCucumber(TestNGCucumberRunner.java:69)
	at cucumber.api.testng.AbstractTestNGCucumberTests.feature(AbstractTestNGCucumberTests.java:21)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:282)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:83)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:114)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:286)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:240)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:121)
Caused by: java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertTrue(Assert.java:52)
	at pageObject.DashBoardPagePO.verifyPage(DashBoardPagePO.java:43)
	at stepDefinition.DashBoardPage.user_is_on_Dashboard_page_with_title(DashBoardPage.java:35)
	at âœ½.And User is on Dashboard page with title "Create Your Board | Trello"(TrelloCreateUser.feature:19)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
            <line>
              <![CDATA[Starting Scenario verify signin page<br/>]]>
            </line>
            <line>
              <![CDATA[**************************************************************************************************<br/>]]>
            </line>
            <line>
              <![CDATA[Ending Scenario verify signin page<br/>]]>
            </line>
            <line>
              <![CDATA[**************************************************************************************************<br/>]]>
            </line>
            <line>
              <![CDATA[Starting Scenario Create User and Dashboard<br/>]]>
            </line>
            <line>
              <![CDATA[**************************************************************************************************<br/>]]>
            </line>
            <line>
              <![CDATA[Ending Scenario Create User and Dashboard<br/>]]>
            </line>
            <line>
              <![CDATA[**************************************************************************************************<br/>]]>
            </line>
          </reporter-output>
        </test-method> <!-- feature -->
        <test-method is-config="true" signature="tearDownClass()[pri:0, instance:runnerFile.RunnerFile@69a10787]" started-at="2019-12-16T13:16:09Z" name="tearDownClass" finished-at="2019-12-16T13:16:10Z" duration-ms="139" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownClass -->
      </class> <!-- runnerFile.RunnerFile -->
    </test> <!-- Sample Test Cases -->
  </suite> <!-- Sample Test Suite -->
</testng-results>
